src/http.js

```diff
@@ -45,8 +45,11 @@ class Http {
     this.defaultHeaders = {};
     this.requestExecutor = httpConfig.requestExecutor;
     this.cacheStore = httpConfig.cacheStore || new MemoryStore();
-    if (httpConfig.cacheMiddleware !== null) {
+    if (httpConfig.cacheMiddleware) {
       this.cacheMiddleware = httpConfig.cacheMiddleware || defaultCacheMiddleware;
+    } else if (httpConfig.cacheMiddleware !== null) {
```

Comment:
you mean `===` here? or just `else` is enough?

Reply:
`httpConfig.cacheMiddleware: null` is for explicitly disabling caching, this branch is intended for cases when cache middleware is not specified in config (is `undefined`)

Reply:
Can you add a comment here?